babel.config.js

```diff
@@ -1,12 +1,23 @@
+function getAliasesFromTsConfig() {
+  const tsConfig = require('./tsconfig.json')
+  const paths = tsConfig.compilerOptions.paths
+  let alias = {}
+  Object.keys(paths).forEach((key) => {
+    alias[key] = `./${paths[key][0]}`
+  })
+
+  return alias
+}
+
 const baseConfig = {
   presets: ['module:metro-react-native-babel-preset'],
   plugins: [
     [
       'module-resolver',
       {
-        alias: {
-          '@helium/crypto': './node_modules/@helium/crypto-react-native',
-        },
+        alias: getAliasesFromTsConfig(),
```

Comment:
this allows us to use `@assets/images/...etc` as an alias. I tried it with `@components` too but typescript wasn't able to track down their definitions, so just leaving it as assets for now

Reply:
Did this remove the `@helium/crypto` one though?